### PR TITLE
IDEMPIERE 5929 - Looping on ZK happening on Exceptions caused by IServerPushCallback.updateUI implementations

### PR DESCRIPTION
**Pull request by MarcosGOFavaretto**
# Refs
[Ticket](https://idempiere.atlassian.net/browse/IDEMPIERE-5929)
[MatterMost](https://mattermost.idempiere.org/idempiere/pl/zawqq7tu53fj8mk1c6kwqhrtie)
# Pull Request Checklist

- [x] My code follows the [code guidelines](https://wiki.idempiere.org/en/Contributing_to_Trunk) of this project
- [x] My code follows the best practices of this project
- [x] I have performed a self-review of my own code
- [x] My code is easy to understand and review. 
- [x] I have checked my code and corrected any misspellings
- [ ] In hard-to-understand areas, I have commented my code.
- [x] My changes generate no new warnings
### Tests
- [x] I have tested the direct scenario that my code is solving
- [x] I checked all collaterals that can be affected by my changes, and tested other potential affected scenarios
- [x] New and existing unit tests pass locally with my changes
- [ ] I have added unit tests that prove my fix is effective or that my feature works
### Documentation
- [ ] I have made corresponding changes to the documentation as follows:
- - [ ] New feature (non-breaking change which adds functionality): I have created the New Feature page in the project wiki explaining the functionality and how to use it. If relevant, I have committed sample data to the core seed to have usable examples in GardenWorld.
- - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected): I have documented the change in a clear way that everyone in the community can understand the impact of the change.
- - [ ] Improvement (improves and existing functionality): This documentation is needed if the improvement changes the way the user interacts with the system or the outcome of a process/task changes. If it is just, for instance, a performance improvement, documentation might not be needed. 
- [ ] The changed/added documentation is in the project wiki (not privately-hosted pdf files or links pointing to a company website) and is complete and self-explanatory.

### CODE REVIEW

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/part/WindowContainer.java [hunk 1]
```diff
@@ -709,35 +709,28 @@ public Tab insertAfter(Tab refTab, Component comp, String title, boolean closeab
...
+		else {
+			firstChild.detach();
+			comp.setParent(refpanel);
+		}
+		
+		if (title != null) 
+		{
+			setTabTitle(title, refTab);
+		}
+		return refTab;
```
Comment (hengsin) [inline org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/part/WindowContainer.java:733]: This change seems unnecessary.

**Review by hengsin — CHANGES_REQUESTED** [review]

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/part/WindowContainer.java (hunk 1 above)
Comment (MarcosGOFavaretto) [reply org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/part/WindowContainer.java:733]: @hengsin about `replace` method?

Comment (hengsin) [reply org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/part/WindowContainer.java:733]: yes, the replace method

Comment (MarcosGOFavaretto) [reply org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/part/WindowContainer.java:741]: @hengsin done!

Comment (hengsin) [reply org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/part/WindowContainer.java:741]: hi @MarcosGOFavaretto , if you are not making any changes to WindowContainer.java, you should just revert all changes to it. As it is, the change set includes many white space changes to WindowContainer.java which is undesirable.

**Review by hengsin — APPROVED** [review]
looks good
